obj/objects.cpp

```diff
@@ -166,16 +170,19 @@ namespace objects
 
     bool operator==(const TriangleObj &left, const TriangleObj &right)
     {
-        std::vector<core::Vec3> vertexVec({left.point1(), left.point2(), left.point3()});
-        std::vector<core::Vec3> refVec({right.point1(), right.point2(), right.point3()});
-
-        for (size_t ind = 0; ind < vertexVec.size(); ++ind)
-        {
-            if (std::find(vertexVec.begin(), vertexVec.end(), refVec.at(ind)) == vertexVec.end())
-            {
-                return false;
-            }
+      // if other triangle has the same points but declared in different order,
+      // they will be still equal.
+      std::vector<core::Vec3> vertexVec(
```

Comment:
You could make this a method in the TriangleObj class. GetPoints() or sth.

Reply:
done